jsonschema-core.xml

```diff
@@ -143,236 +143,43 @@
-                    </t>
-                    <t>
-                        Without this vocabulary or an equivalent one, JSON Schema can only
-                        be applied to a JSON document as a whole.  In most cases, schema
-                        keywords need to be applied to specific object properties or array items.
-                    </t>
-                </section>
-                <section title="Validation">
-                    <t>
-                        This vocabulary describes the structure of a JSON document
-                        (for instance, required properties and length limitations).
-                        Applications can use this information to validate instances (check that
-                        constraints are met), or inform interfaces to collect user input
-                        such that the constraints are satisfied.
-                    </t>
-                    <t>
-                        Validation behaviour and keywords are specified in
-                        <xref target="json-schema-validation">a separate document</xref>.
-                    </t>
-                </section>
-                <section title="Basic Meta-Data">
-                    <t>
-                        A small set of annotation keywords are defined in
-                        <xref target="json-schema-validation">the validation specification</xref>
-                        to allow associating common kinds of meta-data with an instance.
-                    </t>
-                </section>
-                <section title="Hypermedia and Linking">
-                    <t>
-                        JSON Hyper-Schema produces hyperlinks as annotations available for
-                        use with a JSON document.  It supports resolving URI Templates
-                        and describing the resource and data submission formats required
-                        to use an API.
-                    </t>
-                    <t>
-                        Hyper-schema behaviour and keywords are specified in
-                        <xref target="json-hyper-schema">a separate document</xref>.
-                    </t>
-                </section>
-            </section>
+            <t>
+                JSON Schema uses keywords to assert constraints on JSON instances or annotate those
+                instances with additional information.  Additional keywords are used to apply
+                assertions and annotations to more complex JSON data structures, or based on
+                some sort of condition.
+            </t>
+            <t>
+                To facilitate re-use, keywords can be organized into vocabularies.  A vocabulary
+                consists of a list of keywords, together with their syntax and semantics.
+            </t>
+            <t>
+                JSON Schema can be extended either by adding vocabularies, or less formally
```

Comment:
`by adding vocabularies` where? 
..."when defining a new meta-schema,"

Reply:
I see your point here, although I think I have a better solution.  "adding" makes it sound like there is some specific place to which to add the thing.  I don't want to get into meta-schemas in this overview, so instead of "adding vocabularies" let's go with "defining new vocabularies" or "defining additional vocabularies".  And likewise "defining new keywords" or "defining additional keywords" instead of "adding keywords."

BTW when I do the next clarifications I'll make it clear that the thing you do in a meta-schema is **declare** vocabularies, while **defining** vocabularies is (currently) done in specification text.